app/src/androidTest/java/com/github/sdp/ratemyepfl/fragment/navigation/ReviewFragmentTest.kt

```diff
@@ -0,0 +1,33 @@
+package com.github.sdp.ratemyepfl.fragment.navigation
+
+import androidx.fragment.app.testing.launchFragment
+import com.github.sdp.ratemyepfl.R
+import org.junit.Test
+
+
+class ReviewFragmentTest {
+
+    @Test
+    fun loadsCourseFragmentWhenUsersPressesOnCourseTab() {
+        val scenario =
+            launchFragment<ReviewFragment>(themeResId = R.style.Theme_RateMyEPFL).recreate()
+//        onView(withId(R.id.reviewTabLayout)).perform(CustomViewActions.pressesTab(0))
```

Comment:
Is it normal that these tests are commented ?

Reply:
Yes, for some reason, it did not work. I add it in the MainActivityTest and navigate to this state to test it (since it's a filler from now on, and the test will change next week anyway).